### PR TITLE
Sets node and npm version

### PR DESCRIPTION
**Pull request by aksh93**
Makes strict versioning for node and npm to be used

### CODE REVIEW

File: server/.node-version [hunk 1]
```diff
@@ -0,0 +1 @@
+16.4.2
```
Comment (vipulnsward) [inline server/.node-version:1]: Can prefer LTS instead of this?

Comment (Navaneeth-pk) [reply server/.node-version:1]: `Node.js 16 will be the ‘Current’ release for the next 6 months and then promoted to Long-term Support (LTS) in October 2021`

Wouldn't 16 be better or am I missing something? 🤔

Comment (Navaneeth-pk) [reply server/.node-version:1]: `Node.js 14 will remain in long-term support until April 2023` - 14 won't be an issue too.

Comment (aksh93) [reply server/.node-version:1]: I just used the latest stable and had not checked for LTS. Agree, makes sense to stick with version with LTS until.

**Pull request merged**